presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -418,11 +419,6 @@ public static Path createTemporaryPath(ConnectorSession session, HdfsContext con
         String temporaryPrefix = getTemporaryStagingDirectoryPath(session)
                 .replace("${USER}", context.getIdentity().getUser());
 
-        // use relative temporary directory on ViewFS
```

Comment:
why remove this?

Reply:
because someone already implement similar logic above, "getTemporaryStagingDirectoryPath" is doing that, so remove this